composer.json

```diff
@@ -34,5 +34,8 @@
             "phpstan analyze --ansi --configuration=./vendor/kuria/dev-meta/conf/phpstan-src.neon src",
             "phpstan analyze --ansi --configuration=./vendor/kuria/dev-meta/conf/phpstan-tests.neon tests"
         ]
+    },
+    "replace": {
```

Comment:
What's the purpose of this replacement? If these changes are tagged as `1.1.0` then there shouldn't be need for this.

Reply:
mmhh.. you are right. This might no be necessary. Its a left-over from trying to require my fork instead of your version in my project. I'll remove it.